UPDATE: rewrite some of the documentation to the changes made during … (#48)
treeandsea/DroneController: docs/architecture.adoc

****
DroneStateMapper::
Maps the input from `RequestHandler` to expected `DroneState` for the `calculation layer`.

> Is has currently one method called `keyboard` that calculates an future `DroneState` from keyboard input.

Reviewer: ```suggestion
Currently it has one method called `keyboard` that calculates a future `DroneState` from keyboard input.
```
I still think that the keyboard method has nothing to do with a keyboard, as long as it does not read in the keys.
